src/sliding_sync.rs

```diff
@@ -270,6 +330,13 @@ pub enum MatrixRequest {
         room_id: OwnedRoomId,
         typing: bool,
     },
+    /// Spawn an async task to login to the given Matrix homeserver using the given SSO identity provider ID.
+    ///
+    /// This request will disable anymore attempts of spawning SSO server.
```

Comment:
This comment makes it sound like all future attempts will be permanently disabled, which isn't true.
```suggestion
    /// While an SSO request is in flight, the login screen will temporarily prevent the user
    /// from submitting another redundant request, until this request has succeeded or failed.
```